Scripts/DCS-BIOS/doc/developerguide.adoc

```diff
@@ -20,7 +20,7 @@ It is part of the "Administratively Scoped IPv4 Multicast Space", which is to mu
 
 If you cannot or do not want to deal with multicast UDP, you can also make DCS-BIOS send a copy of the export data stream to a unicast address by editing `BIOSConfig.lua`.
 
-You can also use the `socat` or `DCSBIOSBridge`  utility to connect a serial port (and just about anything else that can be read or written) to the DCS-BIOS import and export streams.
+You can also use the `socat` or `DCSBIOSBridge` (https://github.com/DCS-Skunkworks/DCSBIOSBridge)  utility to connect a serial port (and just about anything else that can be read or written) to the DCS-BIOS import and export streams.
```

Comment:
You can use markdown links like so [example](https://www.example.com)

```
[example](https://www.example.com)
```

Reply:
And if you want the monospace text, you can do that too [`example`](https://www.example.com)  
```
[`example`](https://www.example.com) 
```

Reply:
OK will update in next PR. Working on the userguide.